python/ray/tune/commands.py

```diff
@@ -13,9 +11,10 @@
 
 import pandas as pd
 from pandas.api.types import is_string_dtype, is_numeric_dtype
-from ray.tune.util import flatten_dict
 from ray.tune.result import TRAINING_ITERATION, MEAN_ACCURACY, MEAN_LOSS
 from ray.tune.trial import Trial
+from ray.tune.analysis.experiment_analysis import ExperimentAnalysis
+from ray.tune import TuneError
```

Comment:
@andrewztan can you take a look at the changes to this file in this PR and +1 or -1 if looks good?

Reply:
+1 looks good!